package.json

```diff
@@ -130,5 +130,8 @@
   "browserslist": [
     "> 1%",
     "Firefox ESR"
-  ]
+  ],
+  "volta": {
```

Comment:
@nucleogenesis, Looks like this change is causing a build failure with the error

>error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.

Maybe running yarn install and commiting the changes in the yarn lock could solve the issue?